daemon/api_snapshots.go

```diff
@@ -182,9 +182,14 @@ func getSnapshotExport(c *Command, r *http.Request, user *auth.UserState) Respon
 func doSnapshotImport(c *Command, r *http.Request, user *auth.UserState) Response {
 	defer r.Body.Close()
 
+	expectedSize, err := strconv.ParseInt(r.Header.Get("Content-Length"), 10, 64)
```

Comment:
claimedSize?

Reply:
We will stop reading once the size is reached in #9467 so the client better be right about the length :) Or am I missing something ?